e2e/testdata/live-apply/crd-and-cr/resources/Kptfile

```diff
@@ -0,0 +1,8 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: crd-and-cr
+inventory:
+  namespace: crd-and-cr
+  name: crd-and-cr
+  inventoryID: 24a8006de03ceca5ad9c551b1ee467414b6cb5a6-1625960472425460000
```

Comment:
how is this generated ?

Reply:
Will be good if we can add a README.md describing steps for adding/debugging a test.

Reply:
Updated the tests to avoid the auto-generated inventoryIds. Those are not important for the test.

Also added a README.md with the basic information about the kpt live e2e tests.